app/grandchallenge/components/templates/components/componentinterface_list.html

```diff
@@ -59,27 +60,23 @@ <h2>{{ object_type|title }} {{ list_type|title }} Options</h2>
                     <td>{{ interface.description }}</td>
                     <td>{{ interface.get_kind_display }}</td>
                     <td><code>{{ interface.slug }}</code></td>
-                    {% if list_type == list_type_options.INPUT %}
-                        <td>
-                            {% if interface.is_image_kind %}
-                                <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.mha</code>
-                                or
-                                <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
-                            {% else %}
-                                <code>/input/{{ interface.relative_path }}</code>
-                            {% endif %}
-                        </td>
-                    {% elif list_type == list_type_options.OUTPUT %}
+
+                    {% with list_type|lower as url_segment %}
```

Comment:
Undo this refactoring. At this stage, only touch code that is relevant to your functionality.

Reply:
Done